tools/pfif-tools/app/resources/validate_results.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>{{ page_title }}</title>
+  <link rel="stylesheet" type="text/css" href="/static/style.css">
+</head>
+
+<body>
+
+{% if error %}
```

Comment:
Maybe just render a different file on error, so you can avoid a file-spanning if/else block?

Reply:
Done.